src/assets/HTMLAsset.js

```diff
@@ -23,9 +23,12 @@ const ATTRS = {
     'embed'
   ],
   href: ['link', 'a'],
-  poster: ['video']
+  poster: ['video'],
+  content: ['meta']
 };
 
+const META_ASSETS = ['og:image'];
```

Comment:
Based on [The Open Graph protocol](http://ogp.me/) we should include:

- 'og:url'
- 'og:image'
- 'og:image:url'
- 'og:image:secure_url'
- 'og:audio'
- 'og:audio:url'
- 'og:audio:secure_url'
- 'og:video'
- 'og:video:url'
- 'og:video:secure_url'